nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/ui/canvas/items/flow/local-changes-dialog/local-changes-table/local-changes-table.html

```diff
@@ -0,0 +1,80 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<div class="local-changes-table h-full flex flex-col">
+    <div>
+        <div class="value">Displaying {{ filteredCount }} of {{ totalCount }}</div>
+        <form [formGroup]="filterForm">
+            <div class="flex pt-2">
+                <div class="mr-2">
+                    <mat-form-field>
+                        <mat-label>Filter</mat-label>
+                        <input matInput type="text" class="small" formControlName="filterTerm" />
```

Comment:
I know that in current nifi this input field is also located on the left but I think it would be more consistent with the other use cases like this where users can filter a table is to have the filter on the top right. What do you think?

Reply:
Of course as I typed this I see that the summary tables have the filter on the left so it is probably fine.

Reply:
Yeah, it is on the left everywhere else. I'd say we leave it alone.